cmd/tmsUtils.go

```diff
@@ -0,0 +1,293 @@
+package cmd
```

Comment:
Don't put these utils methods in the cmd dir, they belong in the pkg dir. The cmd dir only holds commands, i.e. Piper steps!